start_vscode_bessemer.sh

```diff
@@ -0,0 +1,478 @@
+                echo -e "start_vscode_bessemer.sh version: $VSC_VERSION\n"
+                exit
+                ;;
+                -u|--username)
+                VSC_USERNAME=$2
+                shift
+                shift
+                ;;
+                -n|--numcpus)
+                VSC_CPUS_PER_TASK=$2
+                shift
+                shift
+                ;;
+                -W|--runtime)
+                VSC_RUN_TIME=$2
+                shift
+                shift
+                ;;
+                -m|--memory)
+                VSC_MEM_PER_JOB=$2
+                shift
+                shift
+                ;;
+                -c|--config)
+                VSC_CONFIG_FILE=$2
+                shift
+                shift
+                ;;
+                -g|--numgpu)
+                VSC_NUM_GPU=$2
+                shift
+                shift
+                ;;-p|--partition)
+                VSC_PARTITION_ID=$2
+                shift
+                shift
+                ;;
+                -i|--interval)
+                VSC_WAITING_INTERVAL=$2
+                shift
+                shift
+                ;;
+                -k|--key)
+                VSC_SSH_KEY_PATH=$2
+                shift
+                shift
+                ;;
+                *)
+                echo -e "Warning: ignoring unknown option $1 \n"
+                shift
+                ;;
+        esac
+done
+
+###############################################################################
+# Check configuration options                                                 #
+###############################################################################
+
+# check if user has a configuration file and source it to initialize options
+if [ -f "$VSC_CONFIG_FILE" ]; then
+        echo -e "Found configuration file $VSC_CONFIG_FILE"
+        echo -e "Initializing configuration from file ${VSC_CONFIG_FILE}:"
+        cat "$VSC_CONFIG_FILE"
+        source "$VSC_CONFIG_FILE"
+fi
+
+# check that VSC_USERNAME is not an empty string
+if [ -z "$VSC_USERNAME" ]
+then
+        echo -e "Error: No TUoS username is specified, terminating script\n"
+        display_help
+else
+        echo -e "TUoS username: $VSC_USERNAME"
+fi
+
+# check number of CPU per task
+
+# check if VSC_CPUS_PER_TASK
+if ! [[ "$VSC_CPUS_PER_TASK" =~ ^[0-9c]+$ ]]; then 
+        echo -e "Error: $VSC_CPUS_PER_TASK -> Incorrect format. Please specify number of tasks per node as an integer and try again\n"
+        display_help
+fi
+
+# check if VSC_CPUS_PER_TASK is <= 40
+if [ "$VSC_CPUS_PER_TASK" -gt "40" ]; then
+        echo -e "Error: $VSC_CPUS_PER_TASK -> Larger than 40. No distributed memory supported, therefore the number of cpus per task needs to be smaller or equal to 40\n"
+        display_help
+fi
+
+if [ "$VSC_CPUS_PER_TASK" -gt "0" ]; then
+        echo -e "Requesting $VSC_CPUS_PER_TASK cpus per task for running the code-server"
+        display_help
```

Comment:
This line needs to be removed. No help is needed if the previous checks pass the number of tasks requested is valid. This section should just play back how many were requested.
```suggestion

```

Reply:
Thanks. Fixed